#########################
# http://cvs.gnupg.org/cgi-bin/viewcvs.cgi/gnupg/keyserver/gpgkeys_hkp.c.diff?r1=text&tr1=1.15.2.7&r2=text&tr2=1.15.2.8&diff_format=u
#
# - taviso@gentoo.org 03/12/2003.
#
###################################
===================================================================
RCS file: /cvs/gnupg/gnupg/keyserver/gpgkeys_hkp.c,v
retrieving revision 1.15.2.7
retrieving revision 1.15.2.8
diff -u -r1.15.2.7 -r1.15.2.8
--- gnupg/keyserver/gpgkeys_hkp.c	2003/05/30 04:00:26	1.15.2.7
+++ gnupg/keyserver/gpgkeys_hkp.c	2003/11/27 12:18:20	1.15.2.8
@@ -268,14 +268,14 @@
 
 	  if(gotit)
 	    {
-	      fprintf(output,line);
+	      fputs (line, output);
 	      if(strcmp(line,"-----END PGP PUBLIC KEY BLOCK-----\n")==0)
 		break;
 	    }
 	  else
 	    if(strcmp(line,"-----BEGIN PGP PUBLIC KEY BLOCK-----\n")==0)
 	      {
-		fprintf(output,line);
+		fputs (line, output);
 		gotit=1;
 	      }
 	}
